feat: add troubleshooting section to monitoring and logging (#1626)
nginx/documentation: content/nginxaas-google/monitoring/enable-nginx-logs.md

1. Remove the project ID under **Log Project ID**.
1. Select **Update**.

## Troubleshooting

> To troubleshoot issues viewing logs in Google Cloud Logging, check for **Failed Log Export to Google** events from your NGINXaaS deployment.

Reviewer: ```suggestion
If you can’t see metrics in Google Cloud Monitoring, check your NGINXaaS events for **Failed Log Export to Google**.
```

Author: This is in the logging page, so this is specific to seeing logs in Google Cloud Logging, not metrics

> In the NGINXaaS console,

Reviewer: ```suggestion
In the NGINXaaS console:
```

1. On the navigation menu, select **Events**.
1. Select **Add Filter**.
1. Select **Affected Object** and the name of your NGINXaaS deployment.

> Events are retained for 14 days and then deleted. 

Reviewer 2: ```suggestion
Events are deleted after 14 days. 
```
